src/modules/ekf2/ekf2_main.cpp

```diff
@@ -1970,7 +1979,7 @@ bool Ekf2::blend_gps_data()
 		float sum_of_all_weights = 0.0f;
 
 		// calculate a weighting using the reported speed accuracy
-		float spd_blend_weights[GPS_MAX_RECEIVERS] = {};
+		float spd_blend_weights[GPS_MAX_RECEIVERS] = {0.0f};
```

Comment:
This is wrong, `{}` means "set all elements to `float(0)`" while `{0.0f}` means "set the first element to `0.0f`"

Reply:
Ah, my bad.  Good catch.